ext/fs/ops.rs

```diff
@@ -111,10 +111,7 @@ fn open_options_to_access_kind(open_options: &OpenOptions) -> OpenAccessKind {
 
 #[op2(stack_trace)]
```

Comment:
Can you remove `stack_trace` here too?

Reply:
This one throws though. Is that ok?

Reply:
Yes, `stack_trace` is only used for collecting stack traces for permissions